docs/user-guide/sync-options.md

```diff
@@ -248,6 +248,16 @@ metadata:
     argocd.argoproj.io/sync-options: ServerSideApply=true
 ```
 
+If you want to disable ServerSideApply for a specific resource while it is enabled at the application level,
+the sync-option annotation can be used:
```

Comment:
maybe just clarify that the annotation needs to be added in the specific resource? I know it is implicit but just to make it super obvious? :)